monitor/wait-for

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
```

Comment:
we can symbolic link all wait-for scripts and put the original inside other folder

Reply:
agree, just check potential issues upon physical container separation.